packages/cli/src/Server.ts

```diff
@@ -2595,6 +2595,20 @@ class App {
 			);
 		}
 
+		if (process.env.NODE_ENV !== 'production') {
+			ignoredEndpoints.push('dev-node-description-updated');
+			this.app.post(
+				'/dev-node-description-updated',
```

Comment:
Maybe `/update-node-description` and add a comment to explain what this does and that this is for dev purposes only.